src/nanodbc.h

```diff
@@ -1019,6 +1019,10 @@ class connection
     /// \brief Returns the native ODBC environment handle.
     void* native_env_handle() const;
 
+
+    /// \brief Returns information from the ODBC connection as a string.
+    string_type get_string_info(short info_type) const;
```

Comment:
That is true, however this function is only intended to return string results, it does no coercion if the results are a non string type, and functions signatures cannot differ solely in their return value.